client/coral-framework/components/Slot.js

```diff
@@ -1,16 +1,22 @@
 import React from 'react';
 import cn from 'classnames';
 import styles from './Slot.css';
+import {connect} from 'react-redux';
 import {getSlotElements} from 'coral-framework/helpers/plugins';
 
-export default function Slot ({fill, inline = false, ...rest}) {
+function Slot ({fill, inline = false, plugin_config: {debug, ...config}, ...rest}) {
   return (
-    <div className={cn({[styles.inline]: inline})}>
-      {getSlotElements(fill, rest)}
+    <div className={cn({[styles.inline]: inline, [styles.debug]: debug})}>
+      {getSlotElements(fill, {...config, ...rest})}
```

Comment:
I think the config part shouldn't have the `...`, this would essentially merge the data defined in config with the extra props passed to the slot, while letting the props from the slot to overwrite any config variables.